credhub.html.md.erb

```diff
@@ -0,0 +1,39 @@
+---
+title: CredHub as a Service (CHaaS)
+owner: Services
+---
+
+<strong><%= modified_date %></strong>
+
+## <a id='overview'></a> Overview
+
+CredHub as a Service allows for developers to store credentials in JSON format securely that applications can use. No longer will credentials be written in plain text as an environment variable.
```

Comment:
I'd suggest to change the wording `No longer will credentials be written in plain text as an environment variable.` as it implies that when I use this service, there are no plain text credentials in the environment variable at all. Maybe something like this?
```suggestion
CredHub as a Service allows for developers to store credentials in JSON format securely that applications can use. In comparison to providing credentials with environment variables (e.g. with cf set-env), CredHub credentials are not visible as environment variables reducing the exposure of sensitive information.
```